application/src/main/java/org/opentripplanner/framework/geometry/WgsCoordinate.java

```diff
@@ -163,6 +163,24 @@ public WgsCoordinate roundToApproximate100m() {
     return new WgsCoordinate(lat, lng);
   }
 
+  /**
+   * Compute a fairly accurate distance between two coordinates. Use the fast version in
+   * {@link SphericalDistanceLibrary} if many computations are needed. Return the distance in
+   * meters between the two coordinates.
+   */
+  public double distanceTo(WgsCoordinate other) {
+    return SphericalDistanceLibrary.distance(
+      this.latitude,
+      this.longitude,
+      other.latitude,
+      other.longitude
+    );
+  }
+
+  public boolean isNorthOf(double latitudeBorder) {
```

Comment:
This method seems overly specific to include in the WgsCoordinate IMO, or do you think it will be generally useful?

Reply:
No, I do not think it is useful, but it belongs here anyway - I could probably have used a bounding-box instead. Maybe I will fix it next time around.